runtime/cmake/caches/win_x86.cmake

```diff
@@ -24,6 +24,7 @@ set(J9VM_ARCH_X86 ON CACHE BOOL "")
 set(J9VM_ENV_DATA64 OFF CACHE BOOL "")
 set(J9VM_ENV_LITTLE_ENDIAN ON CACHE BOOL "")
 set(J9VM_ENV_SSE2_SUPPORT_DETECTION ON CACHE BOOL "")
+set(J9VM_GC_SPARSE_HEAP_ALLOCATION OFF CACHE BOOL "")
```

Comment:
not a 64bit spec

Reply:
so we set J9VM_GC_SPARSE_HEAP_ALLOCATION OFF instead of ON?

Reply:
simplest is just remove the line - we did not update any other 32bit for offheap